Wikipedia/Code/UIApplication+WindowWorkarounds.swift

```diff
@@ -0,0 +1,12 @@
+
+import Foundation
+
+extension UIApplication {
+    @objc var workaroundKeyWindow: UIWindow? {
```

Comment:
We can remove `@objc` since we're not using it for the objc class anymore, right?